bot/cogs/tags.py

```diff
@@ -92,9 +92,8 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
         """Show all known tags, a single tag, or run a subcommand."""
         await ctx.invoke(self.get_command, tag_name=tag_name)
 
-    @tags_group.command(name='get', aliases=('show', 'g'))
-    async def get_command(self, ctx: Context, *, tag_name: TagNameConverter = None) -> None:
-        """Get a specified tag, or a list of all tags if no tag is specified."""
+    async def _get_command(self, ctx: Context, tag_name: TagNameConverter = None) -> None:
```

Comment:
Correct the annotations
```suggestion
    async def _get_command(self, ctx: Context, tag_name: str = None) -> bool:
```